From 4f2dd929618f149087249f3c90cc056d5080b1e4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 3 Nov 2016 01:20:33 -0700
Subject: [PATCH] boinc: Add recipe for boinc-client

Do not add prefix/lib to library search path
let sysroot take care of that

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Inappropriate[Cross-compile specific]

---
 m4/ax_check_gl.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/ax_check_gl.m4 b/m4/ax_check_gl.m4
index e26ce5b930..1e60c9266a 100644
--- a/m4/ax_check_gl.m4
+++ b/m4/ax_check_gl.m4
@@ -58,7 +58,7 @@ else
     else
       ax_try_lib="${ax_lib}"
     fi
-    LIBS="-L${prefix}/lib ${ax_try_lib} ${GL_LIBS} ${ax_save_LIBS}"
+    LIBS="${ax_try_lib} ${GL_LIBS} ${ax_save_LIBS}"
     AC_LINK_IFELSE(
     [AC_LANG_PROGRAM([[
 # if HAVE_WINDOWS_H && defined(_WIN32)
